platform/ui/src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -73,7 +74,8 @@ const Thumbnail = ({
             {seriesNumber}
           </div>
           <div className="flex flex-row items-center flex-1">
-            <Icon name="group-layers" className="w-3 mr-2" /> {numInstances}
+            <Icon name={countIcon || 'group-layers'} className="w-3 mr-2" />{' '}
```

Comment:
Would `{`  ${numInstances}`}` work here?

Reply:
Readability... everything on the same line is better AND the two spaces just sitting on their own is awkward... I compare what is current to templating a sentence like this...

`{'what'}{' '}{'a'}{' '}{'strange'}{' '}{'sentence'}`